mp/src/game/server/neo/neo_player.cpp

```diff
@@ -1298,8 +1308,8 @@ void CNEO_Player::ShowDmgInfo(char* infoStr, int infoStrSize)
 		}
 		UserMessageBegin(filter, "ShowMenu");
 		{
-			WRITE_SHORT(static_cast<short>(m_iDmgMenuNextPage ? 3 : 1)); // Amount of options
-			WRITE_CHAR(static_cast<char>(60)); // 60s timeout
+			WRITE_SHORT(static_cast<short>(((m_iDmgMenuNextPage ? 0b11 : 1) | 1 << 8))); // Amount of options
```

Comment:
What does the magic bitwise math do here? Any chance we could have a constant expression for this to make the intent of the passed data more obvious?